Developer.bundle

```diff
@@ -1,5 +1,5 @@
 #######################################
-# Developer file for host development machine
+# Developer file for host development machine #
```

Comment:
This formatting is strange. As per my comments in a previous PR, the `#` should be extended to cover the longer text. And then I suggested just to simplify the titles by having a single `#` in front with no top, bottom and right wrapping.

Reply:
@hkdobrev Strange from a programmatic point of view or in general ?
I would remove all # and just leave these in front of the text.